src/Features/CSharp/Portable/SignatureHelp/CSharpSignatureHelpService.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using Microsoft.CodeAnalysis.CSharp.SignatureHelp.Providers;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.SignatureHelp;
+
+namespace Microsoft.CodeAnalysis.CSharp.SignatureHelp
+{
+    [ExportLanguageServiceFactory(typeof(SignatureHelpService), LanguageNames.CSharp), Shared]
+    internal class CSharpSignatureHelpServiceFactory : ILanguageServiceFactory
+    {
+        public ILanguageService CreateLanguageService(HostLanguageServices languageServices)
+        {
+            return new CSharpSignatureHelpService();
+        }
+    }
+
+    internal class CSharpSignatureHelpService : CommonSignatureHelpService
+    {
+        private readonly ImmutableArray<SignatureHelpProvider> _defaultProviders =
```

Comment:
no.  You don't need the ```=>``` form, but instead the ```{ get; } = ...``` form.  That's just the shorthand for the version you have here.  (Indeed, i'd love to make a refactoring to offer to make this change).